Frontend.Tests/ControllerTests/Projects/BenefitsControllerTests.cs

```diff
@@ -286,6 +286,16 @@ public async void GivenGetByUrnReturnsError_DisplayErrorPage()
                     Assert.Equal("ErrorPage", viewResult.ViewName);
                     Assert.Equal("Project not found", viewModel);
                 }
+                
+                [Fact]
+                public async void GivenUrnAndNoOtherFactors_CreateErrorOnTheView()
+                {
+                    var otherFactors = new List<TransferBenefits.OtherFactor>();
+                    var response = await _subject.OtherFactorsPost("0001", otherFactors, "", "", "");
+                    var viewModel = ControllerTestHelpers.GetViewModelFromResult<BenefitsViewModel>(response);
+                    Assert.True(viewModel.FormErrors.HasErrors);
+                    Assert.True(viewModel.FormErrors.HasErrorForField("otherFactors"));
+                }
```

Comment:
Looking at the requirements for this - it's not that they shouldn't be able to select anything (if they were to select something and deselect it afterwards that would be fine). It's that if they've selected "X other factor", then "X specify this concern further" should be filled in. E.g. Selecting high profile requires you to fill in the "specify this concern further" for that other factor. 

Which does make this slightly more complex, but you could do it with a theory for each other factor at least

Reply:
Ah good point, I'll get it amended